js/atree.js

```diff
@@ -680,20 +680,21 @@ const atree_stats = new (class extends ComputeNode {
                 switch (effect.type) {
                 case 'stat_scaling':
                     if (effect.slider) {
-                        // TODO: handle
-                        const slider_val = interactive_map.get(effect.slider_name).slider.value;
-                        let total = parseInt(slider_val) * effect.scaling[0];
-                        if ('max' in effect && total > effect.max) { total = effect.max; }
-                        if (Array.isArray(effect.output)) {
-                            for (const output of effect.output) {
-                                if (output.type === 'stat') {
-                                    merge_stat(ret_effects, output.name, total);
+                        if ('output' in effect) { // sometimes nodes will modify slider without having effect.
+                            const slider_val = interactive_map.get(effect.slider_name).slider.value;
+                            let total = parseInt(slider_val) * effect.scaling[0];
+                            if ('max' in effect && total > effect.max) { total = effect.max; }
+                            if (Array.isArray(effect.output)) {
+                                for (const output of effect.output) {
+                                    if (output.type === 'stat') {   // TODO: prop
+                                        merge_stat(ret_effects, output.name, total);
+                                    }
                                 }
                             }
-                        }
-                        else {
-                            if (effect.output.type === 'stat') {
-                                merge_stat(ret_effects, effect.output.name, total);
+                            else {
+                                if (effect.output.type === 'stat') {
+                                    merge_stat(ret_effects, effect.output.name, total);
+                                }
```

Comment:
git diff butchered this; i just wrap the whole thing in `if ('output' in effect)` to not try to apply boosts if there are no boosts (slider effect only modified slider)